Test/HUnit/Base.hs

```diff
@@ -2,6 +2,7 @@
 #if MIN_VERSION_base(4,8,1)
 #define HAS_SOURCE_LOCATIONS
 {-# LANGUAGE ImplicitParams #-}
+{-# LANGUAGE ConstrainedClassMethods #-}
```

Comment:
Actually also the `GHC.SrcLoc` removal diff-hunk is needed... 

in any case, the testsuite will fail then for GHC 8.0 if we omit the ErrorCall special handling